solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -275,6 +275,7 @@ The attributes that can be modified are:
 * replicationFactor
 * collection.configName
 * readOnly
+* async
```

Comment:
#161 proposes to document `async` as parameter instead of the attributes list here.